<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArduinoConnect</class>
 <widget class="QWidget" name="ArduinoConnect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>303</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="portLabel">
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="portSelectorComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="baudLabel">
     <property name="text">
      <string>Baud</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="baudSelectorComboBox">
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="detectDevice">
     <property name="toolTip">
      <string>Detect the device connected</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: grergb(241, 241, 241)y;</string>
     </property>
     <property name="text">
      <string>Detect device(s)</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="connectButton">
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: grergb(241, 241, 241)y;</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="IDELabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Arduino IDE</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="IDEExeLabel">
     <property name="text">
      <string>Executable</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QPushButton" name="setIDEButton">
     <property name="text">
      <string>Set IDE</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QPushButton" name="runIDEButton">
     <property name="text">
      <string>Run IDE</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="connectionLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Connection setting</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="IDEPathText">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
